v3/testvectors/go.mod

```diff
@@ -2,6 +2,11 @@ module amazon-s3-encryption-client-go/testvectors
 
 go 1.20
 
+// uncomment this to make local testing easier.
+replace (
```

Comment:
I assume you need to remove this before final release?

Reply:
Discussed offline; I think it's better to just test whatever the testvectors are packaged with since that way we're always testing the latest and we don't have to make multiple commits to get CI to pass for every behavior change/new feature.